src/components/Messaging/MessageBody.js

```diff
@@ -0,0 +1,294 @@
+
+import { MessagingContext } from 'context/MessagingContext.js';
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    messageRow: {
+      display: 'flex',
+    },
+    messageRowRight: {
+      display: 'flex',
+      justifyContent: 'flex-end',
+    },
+    recievedMessage: {
+      position: 'relative',
+      marginLeft: '20px',
+      marginBottom: '10px',
+      padding: '10px',
+      backgroundColor: 'lightGrey',
+      textAlign: 'left',
+      borderRadius: '10px',
+    },
+    sentMessage: {
+      position: 'relative',
+      marginRight: '20px',
+      marginBottom: '10px',
+      padding: '10px',
+      backgroundColor: theme.palette.primary.main,
+      textAlign: 'left',
+      borderRadius: '10px',
+    },
+    messageContent: {
+      padding: 0,
+      margin: 0,
+      wordWrap: 'break-word',
+    },
+    messageTimeStampLeft: {
+      display: 'flex',
+      marginRight: 'auto',
+      alignItems: 'center',
+      color: 'grey',
+    },
+    messageTimeStampRight: {
+      display: 'flex',
+      marginLeft: 'auto',
+      alignItems: 'center',
+      color: 'grey',
+    },
+    displayName: {
+      marginLeft: '20px',
+    },
+    paper: {
+      height: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      flexDirection: 'column',
+    },
+    senderInfo: {
+      padding: '10px',
+      borderBottom: '2px solid black',
+    },
+    messagesBody: {
+      height: '80%',
+      width: '95%',
+      padding: '7.5px',
+      overflowY: 'auto',
+    },
+    senderItem: {
+      padding: theme.spacing(1),
+      margin: theme.spacing(1),
+    },
+    avatar: {
+      width: '5em',
+      height: '5em',
+      [theme.breakpoints.down('md')]: {
+        width: '4em',
+        height: '4em',
+      },
+      [theme.breakpoints.down('sm')]: {
+        width: '2em',
+        height: '2em',
+      },
+    },
+    textInput: {
+      borderTop: '2px solid black',
+    },
+    surveyContent: {
+      color: '#fff',
+    },
+  })
+);
+
+export const AnnounceMessage = ({ message }) => {
+  const { messageRow, recievedMessage, messageContent } = useStyles();
+  return (
+    <div className={messageRow}>
+      {/* <Avatar alt={''} src={message.authorId ? '' : ''}></Avatar> */}
+      <div className={recievedMessage}>
+        <div>
+          <Typography className={messageContent}>{message.body}</Typography>
+        </div>
+      </div>
+      {/* <Grid item className={messageTimeStampRight}>
+          <Typography>{message.composed_at.slice(0, 10)}</Typography>
+        </Grid> */}
+    </div>
+  );
+};
+
+export const SurveyMessage = ({ message }) => {
+  const { messageRowRight, sentMessage, surveyContent } = useStyles();
+
+  return (
+    <div className={messageRowRight}>
+      <Grid item className={sentMessage}>
+        <Typography variant={'h5'} className={surveyContent}>
+          {message.body}
+        </Typography>
+        {message.survey.questions.map((question, i) => (
```

Comment:
would it be cleaner to de-structure this so we don't access too many nested fields? should there also be some validation in case message or survey can be `undefined`?